pom.xml

```diff
@@ -76,6 +76,11 @@
 			<groupId>org.hibernate.validator</groupId>
 			<artifactId>hibernate-validator</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.1.0</version>
+		</dependency>
```

Comment:
This PR contains changes from 2 tasks, add User entity and Pagination, Sorting, Swagger. Please make PR only with relevant changes